traitlets/config/application.py

```diff
@@ -374,7 +374,6 @@ def print_options(self):
             return
         lines = ['Options']
         lines.append('=' * len(lines[0]))
-        lines.append('')
```

Comment:
Can we have the blank lines back? I think it reads clearer when they are there, and I generally think it's always nice to have a blank line after a heading.